config/database.docker.yml

```diff
@@ -1,10 +1,11 @@
-# SQLite Configuration used for Autolab Docker Deployments
-development: &common
-  adapter: sqlite3
-  database: db/db.sqlite3
-  pool: 5
-  timeout: 5000
-
-# SQLite Configuration
+# MySQL Configuration used for Autolab Docker Deployments
 production:
-  <<: *common
+  adapter: mysql2
+  database: autolab_development
+  pool: 5
+  username: user
+  password: password
+  socket: /var/run/mysqld/mysqld.sock
+  host: localhost
```

Comment:
I think this should be mysql instead of localhost, as right now I'm facing this issue when running `make db-migrate`:

```
$ make db-migrate
docker exec -it autolab bash /home/app/webapp/docker/db_migrate.sh
rails aborted!
Mysql2::Error: Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)
/usr/local/rvm/gems/ruby-2.6.6/gems/mysql2-0.4.10/lib/mysql2/client.rb:89:in `connect'
/usr/local/rvm/gems/ruby-2.6.6/gems/mysql2-0.4.10/lib/mysql2/client.rb:89:in `initialize'
```